Dockerfile

```diff
@@ -250,23 +253,23 @@ RUN find $PYTHONHOME -name "*.pyc" -o -name "*.pyo" | xargs rm
 # We don't support compiling extension modules so don't need this directory
 RUN rm -rf $PYTHONHOME/lib/python2.7/config-*-linux-gnu
 
+
 ####### Extra packages that don't make sense to pull down in any other stage ########
 FROM ubuntu:16.04 as extra-packages
 
-ARG TARGET_ARCH
-
 RUN apt update &&\
     apt install -y \
-	  curl \
-	  host \
-	  iproute2 \
-	  netcat \
-	  netcat.openbsd \
-	  realpath \
-	  vim
-
-RUN apt install -y openjdk-8-jre-headless &&\
-    mkdir -p /opt/root &&\
+        curl \
+        host \
+        iproute2 \
+        netcat \
+        netcat.openbsd \
+        openjdk-8-jre-headless \
```

Comment:
Take the change to this extra-packages stage out, my PR #976 will fix this and also requires some changes to Java in general.

Reply:
> Take the change to this extra-packages stage out, my PR #976 will fix this and also requires some changes to Java in general.

Will revert and wait for your changes.